lib/widget/webview/webview.dart

```diff
@@ -98,3 +101,17 @@ List<Map<String, dynamic>> getListOfMap(list) {
   }
   return [];
 }
+
+Map<String, String> parseYamlMap(value) {
+  Map<String, String> result = {};
+  if (value is YamlMap) {
+    YamlMap yamlMap = value;
+    for (var i in yamlMap.entries) {
+      result.addAll({
```

Comment:
if either the key is empty, do NOT add it. You are causing unnecessary bugs here.